test/pageobjects/main.page.js

```diff
@@ -0,0 +1,76 @@
+import Page from './page.js';
+
+class MainPage extends Page {
+    /**
+     * define elements
+     */
+    get postText () { return $('#postform-text')};
+    get droplistExpiration () { return $('.form-group.field-postform-expiration  .select2-selection__arrow')};
+    get droplistHighlighting () {return $("[aria-labelledby='select2-postform-format-container']   .select2-selection__arrow")}
+
+    get postName () { return $('#postform-name')};
+    get btnCreate () {return $('.btn.-big')};
+
+    getExpiration(timecode) {return $("li[data-select2-id$='"+timecode+"']")};
```

Comment:
```
`li[data-select2-id$='${timecode}']`
```

Please check and fix all similar issues.

See more: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals